src/tracks/SampleTrack.cpp

```diff
@@ -352,20 +356,30 @@ void SampleTCOView::contextMenuEvent( QContextMenuEvent * _cme )
 	if( fixedTCOs() == false )
 	{
 		contextMenu.addAction( embed::getIconPixmap( "cancel" ),
-					tr( "Delete (middle mousebutton)" ),
-						this, SLOT( remove() ) );
+					tr( individualTCO
```

Comment:
Can we fix the indention of all the `addAction` calls? All the indention is crazy with unnecessary tabs.